src/utilities/photo_utils.tsx

```diff
@@ -21,13 +22,32 @@ import Metadata from '../types/metadata.type';
  *  timestamp
  * }
  */
+
+
+
+
+// Geolocation data from Web browser
```

Comment:
I suggest:
- Putting the options inside this utility function.
- Calling this function `getCurrentGeolocation`.
- exporting this function for use elsewhere.

This function also needs jsDoc documentation.